docs/source/installation.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Setup and options
+---
+<h2 id="installation">Installation</h2>
+
+To get started with Apollo Link, install the `apollo-link` npm package. This exports everything you need to get started, though there may already be a link with the functionality you wan't on npm.
+
+```bash
+npm install apollo-link --save
+```
+
+Apollo Link has two main exports, the `ApolloLink` interface and the `execute` function. The interface is used to create custom links, compose mutliple links together, and can be extended to support more powerful usecases. The `execute` function is how to use a link and an operation to create a request.
+
+<h2 id="basic-usage">Basic Usage</h2>
+
+Links can be used as a stand-alone client or with most major GraphQL clients.
+
+```js
+import { execute, makePromise } from 'apollo-link';
+import HttpLink from 'apollo-link-http';
+
+const uri = 'http://api.githunt.com/graphql';
+const link = new HttpLink({ uri });
+
+execute(link, operation).subscribe({
+  next: data => console.log(`received data ${data}`),
+  error: error => console.log(`received error ${error}`),
+  complete: () => console.log(`complete`),
+})
+
+//For single execution operations, a Promise can be used
+makePromise((execute(link, operation))
+  .then(data => console.log(`received data ${data}`))
+  .catch(error => console.log(`received error ${error}`))
+```
+
+`execute` accepts a standard GraphQL request with an AST (parsed query string using a library like graphql-tag) and returns an [Observable](https://github.com/zenparsing/zen-observable#api) that allows subscribing.
```

Comment:
> accepts a standard GraphQL request with an AST

I'm not sure if I would know what to pass in to `execute` based on this sentence.